packages/analytics-js-common/src/constants/integrations/client_server_name.js

```diff
@@ -79,6 +79,7 @@ const clientToServerNames = {
   COMMANDBAR: 'CommandBar',
   NINETAILED: 'Ninetailed',
   XPIXEL: 'XPixel',
+  GAINSIGHT_PX: 'Gainsight_PX',
```

Comment:
```suggestion
  GAINSIGHT_PX: 'Gainsight PX',
```
The server only understands the Display name.

Reply:
done